crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3682,11 +3601,26 @@ impl AuthorityPerEpochStore {
             }
 
             SequencedConsensusTransactionKind::External(ConsensusTransaction {
-                kind: ConsensusTransactionKind::UserTransaction(_tx),
+                kind: ConsensusTransactionKind::UserTransaction(tx),
                 ..
             }) => {
-                // TODO(fastpath): implement handling of user transactions from consensus commits.
-                Ok(ConsensusCertificateResult::Ignored)
+                // Safe because transactions are certified by consensus.
+                let tx = VerifiedTransaction::new_unchecked(*tx.clone());
+                // TODO(fastpath): accept position in consensus, after plumbing consensus round, authority index, and transaction index here.
```

Comment:
Does that mean you'll re-introduce the round, authority index & transaction index on the `VerifiedExecutableTransaction::Consensus` ?

Reply:
Yes that is the hope. But if we don't find enough utility for this, maybe we will drop the TODO.